{
	"id": "oval:org.opensuse.security:def:201718207",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2017-18207",
		"affected": {
			"family": "unix",
			"platform": [
				"SUSE Linux Enterprise Micro 5.2"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2017-18207",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-18207",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2017-18207",
				"ref_url": "https://www.suse.com/security/cve/CVE-2017-18207",
				"source": "SUSE CVE"
			},
			{
				"ref_id": "SUSE-CU-2019:706-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-January/006379.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2019:729-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-January/006367.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2019:731-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-January/006368.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2019:733-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-January/006370.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:166-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-May/006793.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:209-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-June/006960.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:215-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-June/006962.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:218-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-June/006963.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:221-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-June/006964.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:32-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-February/006429.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:34-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-February/006436.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:38-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-February/006443.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:544-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-October/007571.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:773-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-December/007979.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2020:93-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-March/006622.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-CU-2021:232-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2021-June/008901.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-IU-2021:605-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2021-August/009236.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2018:0934-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2018-April/003890.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2018:1786-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2018-June/004216.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2018:2040-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2018-July/004297.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2018:2493-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2018-August/004497.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2019:14246-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2019-December/006249.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2020:0114-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-January/006354.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2020:0234-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-January/006406.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2020:0302-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2020-February/006445.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "openSUSE-SU-2018:2126-1",
				"ref_url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/B5SJJLDTHEAEYXTZGWG4F77UDIZS6RYQ/#B5SJJLDTHEAEYXTZGWG4F77UDIZS6RYQ",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "openSUSE-SU-2020:0086-1",
				"ref_url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/SRKGGFVSV7DDWCMAOSO6E3F66U2CF5XR/",
				"source": "SUSE-SU"
			}
		],
		"description": "\n    ** DISPUTED ** The Wave_read._read_fmt_chunk function in Lib/wave.py in Python through 3.6.4 does not ensure a nonzero channel value, which allows attackers to cause a denial of service (divide-by-zero and exception) via a crafted wav format audio file. NOTE: the vendor disputes this issue because Python applications \"need to be prepared to handle a wide variety of exceptions.\"\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Moderate",
			"cve": [
				{
					"text": "CVE-2017-18207 at SUSE",
					"href": "https://www.suse.com/security/cve/CVE-2017-18207/",
					"impact": "low",
					"cvss3": "3.3/CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:L"
				},
				{
					"text": "CVE-2017-18207 at NVD",
					"href": "https://nvd.nist.gov/vuln/detail/CVE-2017-18207",
					"impact": "medium",
					"cvss3": "6.5/CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1083507",
					"href": "https://bugzilla.suse.com/1083507"
				}
			],
			"issued": {
				"date": "2023-05-23"
			},
			"updated": {
				"date": "2024-09-01"
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/o:suse:suse-microos:5.2"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009904671",
						"comment": "libpython3_6m1_0-3.6.15-10.15.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009904672",
						"comment": "python3-3.6.15-10.15.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009904673",
						"comment": "python3-base-3.6.15-10.15.1 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009681300",
				"comment": "SUSE Linux Enterprise Micro 5.2 is installed"
			}
		]
	}
}
